public/doc/swagger-2-v0.0.2.yaml

```diff
@@ -320,6 +347,10 @@ definitions:
           $ref: "#/definitions/InventoryObjectLazy"
         service_parameters_set:
           $ref: "#/definitions/InventoryObjectLazy"
+        source_region:
+          $ref: "#/definitions/InventoryObjectLazy"
```

Comment:
Shouldn't this point to `SourceRegion`?

Reply:
These need to be lazy objects so that we can save an inventory item referencing e.g. a source_region without having that region in the inventory payload.  This is how we can do targeted refresh without all the sparse tree shenanigans that classic MIQ targeted refresh does.